### PR TITLE
Read VaaS config from ConfigMap

### PR DESCRIPTION
**Pull request by AlfredBroda**
Use Kubernetes ConfigMap for storing VaaS access details.

### CODE REVIEW

File: action/common.go [hunk 1]
```diff
@@ -47,19 +53,34 @@ type CommonConfig struct {
...
+		Director:    c.String(FlagDirector),
+		Address:     c.String(FlagAddress),
+		Port:        c.Int(FlagPort),
+		Canary:      c.Bool(FlagCanaryTag),
 	}
 }
+
+// GetSecretFromFile reads a value from provided file
+func (config CommonConfig) GetSecretFromFile(secretFile string) error {
+	if len(secretFile) != 0 {
```
Comment (tomez) [inline action/common.go]: This is unnecessary, `ReadFile()` can handle this.

File: action/deregister.go [hunk 2]
```diff
@@ -27,6 +27,12 @@ func DeregisterCLI(c *cli.Context) error {
 		return errors.New("no VaaS director specified")
 	}
 
+	err := config.GetSecretFromFile(config.VaaSKeyFile)
+	if err == nil {
```
Comment (tomez) [inline action/deregister.go]: This is unusual and confusing. Why not let `GetSecretFromFile` just return error from ioutils?

File: action/deregister.go [hunk 3]
```diff
@@ -27,6 +27,12 @@ func DeregisterCLI(c *cli.Context) error {
...
+		return fmt.Errorf("error reading VaaS secret key: %s", err)
+	}
+	log.Debugf("Register K8s config: %+v\n", config)
+
```
Comment (tomez) [inline action/deregister.go]: If you want to print whole config, mask secret atleast. Preferably with proper config.String() method.

File: action/deregister.go (hunk 2 above)
Comment (AlfredBroda) [reply action/deregister.go]: This is actually wrong, it should check if it's NOT nil.

File: action/deregister.go (hunk 3 above)
Comment (AlfredBroda) [reply action/deregister.go]: No, I'll just get rid of it instead. :)

**Pull request merged**